ui/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
es5 is old.

Reply:
es5 is suitable for a wider range of environments. Higher "target" version  might limit the compatibility of compiled code with older browsers or environments.

Reply:
which might be already blocked by the need to use node Pf-5

Reply:
Changed target to es6